website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md

```diff
@@ -0,0 +1,253 @@
+
+---
+title: "Getting Started with Karpenter on AWS"
```

Comment:
Can we align the link and the title together and with the other getting started guides?

`Getting started with eksctl`

I'm not sure we need to state `with Karpenter` since we're already on the karpenter docs.

Reply:
done